sdk/recoveryservices-backup/Azure.ResourceManager.RecoveryServicesBackup/src/Generated/Extensions/RecoveryServicesBackupExtensions.cs

```diff
@@ -318,129 +306,33 @@ public static async Task<Response<ProtectionIntentResource>> GetProtectionIntent
-        /// <exception cref="ArgumentException"> <paramref name="vaultName"/> is an empty string, and was expected to be non-empty. </exception>
-        /// <exception cref="ArgumentNullException"> <paramref name="vaultName"/> is null. </exception>
-        [ForwardsClientCalls]
-        public static async Task<Response<BackupResourceEncryptionConfigExtendedResource>> GetBackupResourceEncryptionConfigExtendedResourceAsync(this ResourceGroupResource resourceGroupResource, string vaultName, CancellationToken cancellationToken = default)
+        /// <returns> Returns a <see cref="BackupResourceVaultConfigResource" /> object. </returns>
+        public static BackupResourceVaultConfigResource GetBackupResourceVaultConfig(this ResourceGroupResource resourceGroupResource)
         {
-            return await resourceGroupResource.GetBackupResourceEncryptionConfigExtendedResources().GetAsync(vaultName, cancellationToken).ConfigureAwait(false);
+            return GetExtensionClient(resourceGroupResource).GetBackupResourceVaultConfig();
         }
 
-        /// <summary>
-        /// Fetches Vault Encryption config.
-        /// <list type="bullet">
-        /// <item>
-        /// <term>Request Path</term>
-        /// <description>/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.RecoveryServices/vaults/{vaultName}/backupEncryptionConfigs/backupResourceEncryptionConfig</description>
-        /// </item>
-        /// <item>
-        /// <term>Operation Id</term>
-        /// <description>BackupResourceEncryptionConfigs_Get</description>
-        /// </item>
-        /// </list>
-        /// </summary>
+        /// <summary> Gets an object representing a BackupResourceEncryptionConfigExtendedResource along with the instance operations that can be performed on it in the ResourceGroupResource. </summary>
         /// <param name="resourceGroupResource"> The <see cref="ResourceGroupResource" /> instance the method will execute against. </param>
-        /// <param name="vaultName"> The name of the recovery services vault. </param>
-        /// <param name="cancellationToken"> The cancellation token to use. </param>
-        /// <exception cref="ArgumentException"> <paramref name="vaultName"/> is an empty string, and was expected to be non-empty. </exception>
-        /// <exception cref="ArgumentNullException"> <paramref name="vaultName"/> is null. </exception>
-        [ForwardsClientCalls]
-        public static Response<BackupResourceEncryptionConfigExtendedResource> GetBackupResourceEncryptionConfigExtendedResource(this ResourceGroupResource resourceGroupResource, string vaultName, CancellationToken cancellationToken = default)
+        /// <returns> Returns a <see cref="BackupResourceEncryptionConfigExtendedResource" /> object. </returns>
+        public static BackupResourceEncryptionConfigExtendedResource GetBackupResourceEncryptionConfigExtended(this ResourceGroupResource resourceGroupResource)
```

Comment:
same as above

Reply:
As we discussed offline, this resource is not a singleton, change back to common resource.